src/sdk/pynni/nni/networkmorphism_tuner/networkmorphism_tuner.py

```diff
@@ -191,11 +203,13 @@ def init_search(self):
         if self.verbose:
             logger.info("Initialization finished.")
 
+
     def generate(self):
-        """Generate the next neural architecture.
+        """
+        Generate the next neural architecture.
 
-        Returns
-        -------
+        Return
```

Comment:
This should be `Returns`
https://pandas.pydata.org/pandas-docs/stable/development/contributing_docstring.html#section-4-returns-or-yields

Reply:
Fixed.